src/App.tsx

```diff
@@ -1,25 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
```

Comment:
The `getNumbers` function is used but not imported. Ensure that this function is correctly imported from the `utils` file to avoid runtime errors.